ui/components/DataTable.tsx

```diff
@@ -38,10 +55,11 @@ export interface Props {
   fields: Field[];
   /** A list of data that will be iterated through to create the columns described in `fields`. */
   rows?: any[];
-  /** for passing pagination */
-  children?: any;
+  filters?: FilterConfig;
+  dialogOpen?: boolean;
+  checkboxes?: boolean;
```

Comment:
I think the `checkboxes` prop name is a bit misleading. `rows` and `filters` are passed as data objects (lists, configs) and `checkboxes` is just a boolean value. When I looked at it first I expected to see an array of checkboxes data passed instead of just a flag.

It might be worth it to rename this prop to `hasCheckboxes` or `shouldDisplayCheckboxes` to avoid confusion.